### ISSUE TITLE
What is the current status of this project?

### ISSUE THREAD

**Issue by BearOutThere** [bengfarrell/nuimotion#21]
Hello,
it looks like the last update is from a few years ago.  please let us know what your plans for this project are?  is it abandoned?

**Comment by bengfarrell** [comment]
Right, I haven't done anything on this in a few years. The reason why is that I got it working well under the latest version of Node at the time. Unfortunately, it relies on OpenNI for depth sensing and NITE for skeletal tracking. While OpenNI is awesome, open source, and you can still find it, NITE was closed from the beginning. The creators of NITE, Primsense, were bought by Apple a few years ago and at the time of the sale, they shut down Primsense's website so folks are no longer able to download NITE.

If this node module were to continue, the obvious task at hand would be to update for the latest C++ implementation in the latest version of Node, but even if that were to happen, I'd still be relying on NITE. 

So, yes, I've moved onto other things, because at this point most of what I did here would need to be scrapped.
